.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+ ## What does it do?
+ >Explain what does this PR do in no more than 2 lines
+
+## Tasks to be completed in this PR
+> List all tasks that you have completed and remaining in this PR
+
+## How is it tested locally?
```

Comment:
Codacy found an issue: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=5869579)